### PR TITLE
TV-SHOWS-DOM_PROJECT

### PR DESCRIPTION
**Pull request by Taslima26**
Level 350 completed.
Few gui issues I need to work on .Apart from that functionality works fine.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,19 +3,50 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | My Name Taslima (My Github user-name Taslima26)</title>
```
Comment (azadaafo) [inline index.html:6]: Great work Tas, well done. I think there is no need to say My Github user-name Taslima26. only Taslima26 is enough. 
code wise, nothing I can add really, way better than me. keep up the good work

**Comment by Taslima26** [comment]
Thank you Azad. Let me know if you need any help .
Thank you .

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: script.js [hunk 2]
```diff
@@ -1,12 +1,193 @@
...
+    displayEpisodes(episodesData);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+const loadShows = async () => {
+  try {
+    const res = await fetch('  http://api.tvmaze.com/shows/1/episodes');
+    //How do I  get show id here  everytime user select item??
```
Comment (Dedekind561) [inline script.js]: Not sure what you mean here Tas - did you send me a message about this in Slack ?

File: script.js [hunk 3]
```diff
@@ -1,12 +1,193 @@
...
+let episodesData = [];
+let showsData = [];
+
+
+ 
+const loadEpisodes = async () => {
+  try {
+    const res = await fetch('https://api.tvmaze.com/shows/82/episodes');
+    
+    episodesData = await res.json();
```
Comment (Dedekind561) [inline script.js]: Good variable name here 👍

File: script.js [hunk 4]
```diff
@@ -1,12 +1,193 @@
...
+}
+
+
+
+//const allEpisodesDetails = setup();
+searchBar.addEventListener('keyup', (e) => {
+
+  const searchString = e.target.value.toLowerCase();
+  const filteredMovies = episodesData.filter((item) => {
+    return (
```
Comment (Dedekind561) [inline script.js]: Excellent use of `filter` here - your variable names are clear and I can understand the logic straight away when I read it

File: script.js [hunk 5]
```diff
@@ -1,12 +1,193 @@
...
+      item.name.toLowerCase().includes(searchString) ||
+      item.summary.toLowerCase().includes(searchString)
+    );
+  });
+  displayEpisodes(filteredMovies);
+});
+
+function displayShows(Shows) {
```
Comment (Dedekind561) [inline script.js]: Avoid upper case variable names unless you are defining a `class`

File: script.js [hunk 6]
```diff
@@ -1,12 +1,193 @@
...
+  let displayShow = Shows.map((item) => {
+    return `<article class='movie-item'>
```
Comment (Dedekind561) [inline script.js:105]: I like the fact you're using a template literal here - it works really well 😄

File: script.js [hunk 7]
```diff
@@ -1,12 +1,193 @@
...
+     });
+
+     displayEpisode = displayEpisode.join(" ");
+  
+     sectionCenter.innerHTML = displayEpisode;
+}
+//create select box for episodes
+getAllEpisodes().forEach((item) => {
+
+  let opt = 'S0' + item.season.toString() + 'E0' + item.number.toString() + '-' + item.name;
```
Comment (Dedekind561) [inline script.js]: Could also consider using a template literal here too

File: script.js [hunk 8]
```diff
@@ -1,12 +1,193 @@
...
+  selectEpisodes.appendChild(el);
+
+});
+   
+//select box logic 
+  
+
+  document.querySelector("#select-episode").addEventListener('change', (e) => {
+    const selectString = e.target.value;
+    console.log(selectString);
```
Comment (Dedekind561) [inline script.js]: Be sure to remove `console.logs` when from your code when you've finished with them

File: script.js [hunk 9]
```diff
@@ -1,12 +1,193 @@
...
+   
+  });
+  displayShow = displayShow.join('');
+  sectionCenter.innerHTML = displayShow;
 }
 
-function makePageForEpisodes(episodeList) {
+
+
+function displayEpisodes(Movies) {
```
Comment (Dedekind561) [inline script.js]: Great function name here - something with a very clear and meaningful name

File: script.js [hunk 10]
```diff
@@ -1,12 +1,193 @@
...
+   
+//select box logic 
+  
+
+  document.querySelector("#select-episode").addEventListener('change', (e) => {
+    const selectString = e.target.value;
+    console.log(selectString);
+    const selectedMovies = episodesData.filter((item) => {
+      return (
+        'S0' + item.season.toString() + 'E0' + item.number.toString() + '-' + item.name===selectString
```
Comment (Dedekind561) [inline script.js]:
```suggestion
        'S0' + item.season.toString() + 'E0' + item.number.toString() + '-' + item.name === selectString
```

File: script.js (hunk 7 above)
Comment (Taslima26) [reply script.js]: sure I can do that.Thank you

**Pull request closed without merging**